source/Paragraphs.h

```diff
@@ -0,0 +1,59 @@
+/* Paragraphs.h
+Copyright (c) 2024 by an anonymous author
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef PARAGRAPHS_H_
+#define PARAGRAPHS_H_
+
+#include "ConditionSet.h"
+
+#include <algorithm>
+#include <string>
+#include <vector>
+
+class ConditionsStore;
+class DataNode;
+
+class Paragraphs {
```

Comment:
```suggestion


class Paragraphs {
```
Need a comment on purpose of class

Reply:
How's this one?

> // Stores a list of description paragraphs, and a condition under which each should be shown.
> // See the planet and spaceport description code for examples.